main/src/com/google/refine/model/recon/StandardReconConfig.java

```diff
@@ -154,6 +165,9 @@ public String toString() {
     @JsonProperty("limit")
     final private int limit;
 
+    // initialized lazily
+    private CloseableHttpClient httpClient = null;
```

Comment:
My intention was to try to avoid any overhead there, since this class is deserialized quite often (as soon as you have a reconciled column in the project, it will get saved / restored every time you save the ColumnModel)